README.md

```diff
@@ -11,9 +11,10 @@ Install [Taurus](https://gettaurus.org/docs/Installation/)
 
 ### content-api-crud
 
+`run-config.yml` runs a all CRUD actions on content-api.
```

Comment:
What is bzt? Do we need to install some special software? Would you please add that to the readme?

Reply:
yes, setup instructions are in the existing readme